test/Window/Cursor.test.cpp

```diff
@@ -10,60 +10,38 @@ TEST_CASE("[Window] sf::Cursor", runDisplayTests())
 {
     SECTION("Type traits")
     {
+        STATIC_CHECK(!std::is_default_constructible_v<sf::Cursor>);
         STATIC_CHECK(!std::is_copy_constructible_v<sf::Cursor>);
         STATIC_CHECK(!std::is_copy_assignable_v<sf::Cursor>);
         STATIC_CHECK(std::is_nothrow_move_constructible_v<sf::Cursor>);
         STATIC_CHECK(std::is_nothrow_move_assignable_v<sf::Cursor>);
     }
 
-    SECTION("Construction")
-    {
-        const sf::Cursor cursor;
-    }
-
-    SECTION("Move semantics")
-    {
-        SECTION("Construction")
-        {
-            sf::Cursor       movedCursor;
-            const sf::Cursor cursor(std::move(movedCursor));
-        }
-
-        SECTION("Assignment")
-        {
-            sf::Cursor movedCursor;
-            sf::Cursor cursor;
-            cursor = std::move(movedCursor);
-        }
```

Comment:
Does that mean `sf::Cursor` is no longer movable? Do we always create a copy now?

Reply:
https://github.com/SFML/SFML/blob/7234fc149bf21b12fbb10deec73025fdfec07808/test/Window/Cursor.test.cpp#L15-L16

It's still moveable.

I removed these tests because we use `= default` implementations of move operations and by convention in SFML we don't write tests for defaulted operations. Instead of rewriting these tests for the new API I just removed them. They don't add value.